app/views/pages/home.html.erb

```diff
@@ -1,33 +1,44 @@
 <div class="panel panel-primary">
   <div class="panel-heading">
-    <h3 class="panel-title">Ratttlesnake Ramble 2020 Information</h3>
+    <h3 class="panel-title"><%= "Rattlesnake Ramble #{current_full_course_edition.year} Information" %></h3>
   </div>
   <div class="table">
     <table class="table">
       <tr>
-        <td>When</td>
-        <td>Saturday, September 11, 2021, <b> New Start Time! </b>7:45 AM Kids' Wave, 7:47 AM Men, 8:00 AM Kids</td>
+        <td>Date</td>
+        <td><%= l current_full_course_edition.date, format: :long %></td>
```

Comment:
What's this 'l'?

Reply:
That's a built-in shortcut for the `::I18n::localize` method. It takes a date or datetime object and formats it to a string depending on the specified format.

Reply:
I just updated this line to use a better format. The one I used before (`:long') is built into Rails and gives us something that looks like "September 11, 2021". I changed it to `:long_with_day` and then defined that in `config/locales/en.yml` to include the day of week as well, so now this will look like "Saturday, September 11, 2021".